### PR TITLE
[Runtime] Log when using default grace period for runtime resources deletion

### PR DESCRIPTION
**Pull request by alonmr**

### CODE REVIEW

File: mlrun/db/httpdb.py [hunk 1]
```diff
@@ -950,12 +950,16 @@ def delete_runtime_resources(
         :param force: Force deletion - delete the runtime resource even if it's not in terminal state or if the grace
             period didn't pass.
         :param grace_period: Grace period given to the runtime resource before they are actually removed, counted from
-            the moment they moved to terminal state.
+            the moment they moved to terminal state (default is 4 hours).
```
Comment (Tankilevitch) [inline mlrun/db/httpdb.py:953]: this is problematic to rely on the default value of the configuration, as the configuration can be easily changed on BE while running and this comment won't be relevant anymore. I you want to be accurate, then pass that param through the client-spec.

**Review by Tankilevitch — CHANGES_REQUESTED** [review]

File: mlrun/db/httpdb.py [hunk 2]
```diff
@@ -950,13 +950,17 @@ def delete_runtime_resources(
         :param force: Force deletion - delete the runtime resource even if it's not in terminal state or if the grace
             period didn't pass.
         :param grace_period: Grace period given to the runtime resource before they are actually removed, counted from
-            the moment they moved to terminal state.
+            the moment they moved to terminal state (defaults to mlrun.mlconf.runtime_resources_deletion_grace_period).
```
Comment (Tankilevitch) [inline mlrun/db/httpdb.py:953]: this is not accurate as well, as the `runtime_resources_deletion_grace_period` is managed by the API and not by the client.

Comment (Tankilevitch) [reply mlrun/db/httpdb.py:953]: maybe needs to be passed through the client-spec

Comment (alonmr) [reply mlrun/db/httpdb.py:953]: it is enriched on the server side only on the periodic cleanup

**Review by Tankilevitch — APPROVED** [review]

**Pull request merged**